### PR TITLE
RES-007-component-with-navigation

### PR DESCRIPTION
**Pull request by RafalPe**
Add a navigation bar for customers

### CODE REVIEW

File: src/components/NavigationBar/NavigationBar.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const NavigationBar = () => {
+    return(
+        <Navbar className=" d-flex nav-background">
+            <Col>
+                <Nav className="d-none d-md-flex align-items-center" > 
+                    <Image className="mx-4 my-1" src={RestaurantLogo} alt="restaurant logo"></Image>
+                    <Nav.Link className="categories-text mx-3 p-4 active-page"> 
+                        <NavLink exact to='/' className="text-decoration-none text-white" activeClassName="active">Home</NavLink>
+                    </Nav.Link>
```
Comment (lagdush) [inline src/components/NavigationBar/NavigationBar.js]: You can make a separate component from it and give it props.

File: src/components/NavigationBar/NavigationBar.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                </Nav>
+                <Nav className=" justify-content-end d-md-none d-flex align-items-center" >
+                    <Image className="restaurant-logo my-1 mx-3" src={RestaurantLogo} alt="restaurant logo"></Image>
+                    <Container fluid className="d-flex justify-content-end nav-container active-page">
+                        <NavLink exact to='/' className="text-decoration-none text-white" activeClassName="active">
+                            <Button className="bg-transparent shadow-none border-none">
+                                <Image src={HomeIcon}></Image>
+                            </Button>
+                        </NavLink>
```
Comment (lagdush) [inline src/components/NavigationBar/NavigationBar.js]: You can make a separate component from it and give it props.

**Review by lagdush — CHANGES_REQUESTED** [review]

File: src/components/NavigationBar/NavigationBar.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                            <Button className="bg-transparent shadow-none border-none">
+                                <Image src={HomeIcon}></Image>
+                            </Button>
+                        </NavLink>  
+                        <NavLink exact to='/menu' className="text-decoration-none text-white" activeClassName="active">
+                            <Button className="bg-transparent shadow-none border-none">
+                                <Image src={MenuIcon}></Image>
+                            </Button>
+                        </NavLink>
+                        <NavLink exact to='/contact' className="text-decoration-none text-white" activeClassName="active">
```
Comment (lagdush) [inline src/components/NavigationBar/NavigationBar.js]: See what errors you get in the inspector and please fix them.
Zrzut ekranu 2022-01-22 o 10 19 53

File: src/components/ContactPage/ContactPage.js [hunk 4]
```diff
@@ -1,5 +1,13 @@
+import React from "react";
+import NavigationBar from "../NavigationBar/NavigationBar";
+
 const ContactPage = () => {
-  return <div>ContactPage</div>;
+ return (
+    <div>
+      <NavigationBar />
```
Comment (lagdush) [inline src/components/ContactPage/ContactPage.js]: Everything looks cool and we can almost merge it. You still have to handle the navigation nicely. It's not good to add a navigation component to every view. Try to fix it. If you can't make it by tomorrow, talk to me.

**Review by lagdush — CHANGES_REQUESTED** [review]

File: src/components/NavigationBar/NavigationBar.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+                        name="Home"
+                    />
+                    <Page 
+                        source="/menu"
+                        name="Menu"
+                    />
+                    <Page 
+                        source="/contact"
+                        name="Contact"
+                    />
```
Comment (lagdush) [inline src/components/NavigationBar/NavigationBar.js]: you can map something like this

File: src/components/NavigationBar/Pages.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Page from "./Page";
+
+const Pages = (props) => {
+    const pages =[
+        {source: "/", name: "Home"},
+        {source: "/menu", name: "Menu"},
+        {source: "/contact", name: "Contact"}
+    ];
```
Comment (lagdush) [inline src/components/NavigationBar/Pages.js]: You don't use these props anywhere, so you can remove them.

File: src/components/NavigationBar/PagesOnMobile.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+import PageOnMobile from "./PageOnMobile";
+import HomeIcon from "../../images/home-icon.png"
+import MenuIcon from "../../images/menu-icon.png"
+import ContactIcon from "../../images/contact-icon.png"
+
+const PagesOnMobile = (props) => {
+    const pagesOnMobile =[
+        {source: "/", icon: HomeIcon},
+        {source: "/menu", icon: MenuIcon},
+        {source: "/contact", icon: ContactIcon}
```
Comment (lagdush) [inline src/components/NavigationBar/PagesOnMobile.js]: You don't use these props anywhere, so you can remove them.

**Review by lagdush — CHANGES_REQUESTED** [review]
One last small change and it will be ok.

**Review by lagdush — APPROVED** [review]
Well done.

**Review by EMEZYK — APPROVED** [review]

**Pull request merged**